main/main.go

```diff
@@ -542,46 +550,39 @@ func (difftool *xdcrDiffTool) waitForDuration(sourceDcpDriver, targetDcpDriver *
 
 func (difftool *xdcrDiffTool) retrieveReplicationSpecInfo() error {
 	// CBAUTH has already been setup
-	rcMap, err := difftool.remoteClusterSvc.RemoteClusters()
+	var err error
+	difftool.specifiedRef, err = difftool.remoteClusterSvc.RemoteClusterByRefName(options.remoteClusterName, false /*refresh*/)
 	if err != nil {
 		difftool.logger.Errorf("Error retrieving remote clusters: %v\n", err)
 		return err
 	}
 
+	if options.targetUsername != "" && options.targetPassword != "" {
+		difftool.logger.Infof("Target username and password were specified. Overriding them in the retrieved remote cluster reference")
+		difftool.specifiedRef.UserName_ = options.targetUsername
+		difftool.specifiedRef.Password_ = options.targetPassword
```

Comment:
these may not be sufficient. remoteClusterRef may have encryption set to half or full.  if username and password are explicitly passed in, maybe the intention is to use SASL? if this is the case encryption mode needs to be changed to none.

Reply:
Sure. Force the auth to plain.